scss/2-tools/_interactive-elements.scss

```diff
@@ -1,6 +1,4 @@
 @mixin cads-interactive-element() {
-  @include cads-transition-animation();
```

Comment:
Removed this as the places that called `cads-interactive-element` all then disabled `transition` afterwards. So this seems to be unintentional. Or better to require explicit use when needed.